roxctl/declarativeconfig/create/access_scope.go

```diff
@@ -108,8 +130,16 @@ func (a *accessScopeCmd) PrintYAML() error {
 	if err := lint.Lint(yamlOut.Bytes()); err != nil {
 		return errors.Wrap(err, "linting the YAML output")
 	}
-	_, err := a.env.InputOutput().Out().Write(yamlOut.Bytes())
-	return errors.Wrap(err, "writing the YAML output")
+	if a.writeToK8sObject {
```

Comment:
nit: I don't see much profit in introducing such field since it is used only once